src/platform_cfg.cpp

```diff
@@ -618,6 +629,18 @@ void CPlatformYamlInfo::Dump(FILE *fd){
         fprintf(fd," m_tx_desc    :  %d \n",(int)m_tx_desc);
     }
 
+    if (m_latency_measurement.length()) {
```

Comment:
Ditto: .empty()

Reply:
IMO `.lentgh()` is better than `! .empty()`